client/components/paginated-survey/components/question-step.tsx

```diff
@@ -0,0 +1,66 @@
+import { StepContainer } from '@automattic/onboarding';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { Question, QuestionType } from '../types';
+import QuestionMultipleOptions from './question-multiple-selection';
+import QuestionSingleOption from './question-single-selection';
+import './style.scss';
+
+const questionsMap = {
```

Comment:
Was a little confused when I read this variable name at first, perhaps something like `questionTypeComponentMap` for clarity?

Reply:
Agreed, your suggestion looks better.